Anamnesis/Actor/Pages/CharacterPage.xaml

```diff
@@ -130,13 +130,32 @@
 			</Grid>
 		</GroupBox>
 
-		<!-- Warning for making sure the actor can be edited. (Out of GPOSE when not using Brio or Penumbra integrations) -->
+		<!-- Warning messages indicating that the actor's appearance cannot be edited. -->
+		<XivToolsWpf:InfoControl
+			Key="Character_WarningGposeWorldPosFrozen"
+			Grid.Column="0"
+			Grid.ColumnSpan="2"
+			Margin="12">
+			<XivToolsWpf:InfoControl.Visibility>
+				<MultiBinding Converter="{StaticResource MultiBoolAndToVisibility}">
+					<Binding Path="Actor.CanRefresh" Converter="{StaticResource !B}"/>
```

Comment:
Will this respect if a Penumbra redraw is used without Brios presence? Im rather sure that if it uses Penumbra redraw it should not trigger this bug if Brio isnt present, right? So if we disable it here we may rob some people of this ability despite it being perfectly fine.

Reply:
Penumbra redraw also has this limitation: https://github.com/imchillin/Anamnesis/blob/bdb2f5fef9ef295ffcf37e42b81df856cfe48f50/Anamnesis/Actor/Refresh/PenumbraActorRefresher.cs#L23.

Reply:
![image](https://github.com/user-attachments/assets/b58d2b7d-fa31-40a5-a22b-e76905c92227)
Case in point, it does. Damn. I guess I just never ever ever used frozen Position. In that case never mind me lol...